### ISSUE TITLE
java.lang.IllegalArgumentException: Value too long

### ISSUE THREAD

**Issue by Patrick-Poitras** [TurboVNC/turbovnc#410]
I'm getting this error when trying to connect to a new server.

java.lang.IllegalArgumentException: Value too long: 192.168.45.16:30906,192.168.45.16:31840,...

With ... being the entirety of the values I've used to connect before, separated by commas. Seems like Java is balking at how many connections I've had before when trying to parse the history.

Details: Windows 10, tested on 3.0.3

**Comment by Patrick-Poitras** [comment]
![image](https://github.com/TurboVNC/turbovnc/assets/42386710/0ed3954f-d6cb-49c5-9289-9f7bb0c1a9ac)

**Comment by Patrick-Poitras** [comment]
Workaround found: I went into the settings and cleared the connection history. This fixed the problem, at the expense of clearing my history.

**Comment by dcommander** [comment]
That's really weird. So the viewer appeared to be trying to use a comma-separated list of previous connections as the server name? I can't imagine why it would have done that. Any further information you can provide that may help me reproduce the issue would be appreciated.

**Comment by dcommander** [comment]
Also, assuming you got a stack trace when the failure occurred, can you post the stack trace?

**Comment by Patrick-Poitras** [comment]
I got nothing more. The error message was a popup window from the viewer that chained the addresses I had previously connected to, separated by commas. 

I am no longer able to reproduce due to clearing the history, but that also fixed the problem, so we've got a workaround. My guess is that the dropdown menu is serialized into a comma separated list at some point and then sent through some API that chokes on the amount of data.
